packages/dx-react-scheduler/docs/reference/day-view.md

```diff
@@ -136,6 +136,9 @@ Field | Type | Description
 ------|------|------------
 startDate? | Date | Specifies the cell a start time.
```

Comment:
"the cell a start time" the a?

Reply:
It's a typo.
```suggestion
startDate? | Date | Specifies the cell's start time.
```

@AryamnovEugeniy Could you check if there are more entries of this and fix them?

Reply:
I will